testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPoliciesLoadUpdateTest.java

```diff
@@ -161,8 +161,7 @@ public void testDuplicatedProfiles() throws Exception {
         // load profiles
```

Comment:
Comment unrelated to this line.

It seems `augment` is still mentioned at several places in this file. I wonder why the tests passed.

Reply:
Yes, it is expected. There are tests, which are trying to upload the JSON with invalid configuration and they tests that upload is not successful. The invalid configuration is JSON string, which contains those invalid configuration fields (including `augment` field, which I still kept in this String).

Reply:
Thanks for the clarification! I thought it's a mistake that the `augment` fields were left unchanged. LGTM then.

Reply:
There are 3 test method updating client profiles including "client-authns-augment" and "is-augment" and these are expected to errors due to other reason that does not relate to "client-authns-augment" and "is-augment". These fields are originally valid fields but become invalid fields. However, these invalid fields are skipped so that these fields does not cause errors.